testing/src/test/java/org/oppia/android/testing/data/AsyncResultSubjectTest.kt

```diff
@@ -5,9 +5,12 @@ import com.google.common.truth.Truth.assertThat
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.oppia.android.util.data.AsyncResult
+import org.robolectric.annotation.Config
+import java.io.FileNotFoundException
 
 /** Tests for [AsyncResultSubject]. */
 @RunWith(AndroidJUnit4::class)
```

Comment:
Is there an error running with JUnit4?
```suggestion
@RunWith(JUnit4::class)
```